proto/parca/query/v1alpha1/query.proto

```diff
@@ -224,6 +224,9 @@ message QueryRequest {
 
   // filter_query is the query string to filter the profile samples
   optional string filter_query = 6;
+
+  // disable_trimming disables trimming of insignificant nodes in the report
+  optional bool disable_trimming = 7;
```

Comment:
if we allow disabling it in the API at all, then I think we should allow passing the exact parameters to trim with

Reply:
Yeah right, you mean the `NodeCutOffFraction` value, right?

Reply:
This is done.

Reply:
What if we always set the `node_trim_threshold` in the UI and if it's unset (or the default value of 0.0) then it's disabled?

Reply:
Yes, that sounds simpler. Will do that.

Reply:
This is done.